app/api/schema/event_statistics.py

```diff
@@ -37,7 +37,7 @@ def sessions_draft_count(self, obj):
         return Session.query.filter_by(event_id=obj.id, state='draft').count()
 
     def sessions_submitted_count(self, obj):
-        return Session.query.filter_by(event_id=obj.id, state='submitted').count()
+        return Session.query.filter_by(event_id=obj.id).count()
```

Comment:
Filter deleted sessions out

Reply:
Won't the deleted session be removed from the table?

Reply:
NVM. Got it. Have to check `deleted_at` :).